### PR TITLE
Docs install cleanup

### PR DESCRIPTION
**Pull request by kickoke**
This is basically a docs housekeeping pull request: 
- Reworked the MacOS installation page
- Updated instances of the kickstart script
- Deleted legacy information: SOCKS5 support, libwebsockets, libmosquitto

There are still files with the old kickstart script. However these were located in the backends folder. Backends will be removed, soon, so I didn't see a point in updating them

### CODE REVIEW

File: claim/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ When coming from  [Nodes view page](https://learn.netdata.cloud/docs/cloud/visua
 If you want to connect a node that is running on a Linux environment, the script that will be provided to you by Netdata Cloud is the [kickstart](/packaging/installer/README.md#automatic-one-line-installation-script) which will install the Netdata Agent on your node, if it isn't already installed, and connect the node to Netdata Cloud. It should be similar to:
 
 ```
-bash <(curl -Ss https://my-netdata.io/kickstart.sh) --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
+wget -O ./kickstart.sh https://my-netdata.io/kickstart.sh && sh ./kickstart.sh --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
```
Comment (ilyam8) [inline claim/README.md:76]: "https://app.netdata.cloud" is the [default](https://github.com/netdata/netdata/blob/162f2b62b2dc6e38e8cd41c63bd36998b1c180fd/packaging/installer/kickstart.sh#L24), so `--claim-url` is not really needed.

Comment (kickoke) [reply claim/README.md:76]: Good to know. I've included it as the other examples all had it. 
Is it even possible to have a custom claim url at the moment?
@ilyam8

Comment (ilyam8) [reply claim/README.md:76]: Not really (if we don't count us testing not in prod env). 

>  I've included it as the other examples all had it.

I see, ok.

**Review by DShreve2 — APPROVED** [review]

**Comment by Ferroin** [comment]
Merging without waiting for CI as this has received the required approvals and only changes documentation.

**Comment by kickoke** [comment]
@Ferroin @DShreve2 
In all of the stress and the rush of the last few days I completely forgot that what I *really* wanted to do was place the [interactive installation component](https://learn.netdata.cloud/docs/get-started#install-on-linux-with-one-line-installer) everywhere where we currently use markdown. 
This way, we only have to make amendments once. :) 
So I restored the branch and I will update all instances. 
Very sorry for the back and forth.

**Comment by ilyam8** [comment]
Speaking about the component. 

<details>
<summary>is it expected to be so big?</summary>

Screenshot 2022-02-03 at 17 06 39

</details>

**Comment by kickoke** [comment]
I can see if I can tweak the css. It does look excessively big.

**Pull request merged**